mobile/lib/quiz/show/screens/quiz_page.dart

```diff
@@ -1,20 +1,33 @@
+import 'dart:convert';
+
 import 'package:flutter/material.dart';
+import 'package:flutter_secure_storage/flutter_secure_storage.dart';
 import 'package:go_router/go_router.dart';
 import 'package:mobile/common/providers/item_list_provider.dart';
+import 'package:mobile/search_engine/dto/item_dto.dart';
+import 'package:mobile/search_engine/dto/results_dto.dart';
 import 'package:provider/provider.dart';
 
 import '../widgets/header_quiz_page.dart';
 import '../widgets/heart_favourite.dart';
 import '../../common/widgets/quiz_button.dart';
 import '../widgets/quiz_page_text_container.dart';
 
-class QuizPage extends StatelessWidget {
+class QuizPage extends StatefulWidget {
```

Comment:
dlatego używamy stateful widgetu? tutaj nic nie jest updatowane, to jest tylko strona co przekierowywuje dalej